es710x/src/main/scala/tech/beshu/ror/es/handler/request/context/types/ReflectionBasedIndicesEsRequestContext.scala

```diff
@@ -57,7 +57,7 @@ class ReflectionBasedIndicesEsRequestContext private(actionRequest: ActionReques
     // Optimistic reflection attempt
     ReflecUtils.setIndices(
       actionRequest,
-      Sets.newHashSet("index", "indices", "setIndex", "setIndices"),
+      Sets.newHashSet("index", "indices", "setIndex", "setIndices", "name", "getName"),
```

Comment:
we should handle all requests from the `org.elasticsearch.action.datastreams` package explicitly. They should be handled before this generic index-based request handling. Let's treat the name of the stream as the index name at the moment. It should be safe enough but it'll unblock the data stream API (currently clients cannot use it because of the bug)